barcode/codex.py

```diff
@@ -271,5 +271,72 @@ def get_fullcode(self):
         return super().get_fullcode()[1:]
 
 
+class MSI(Barcode):
+    """Initializes a new MSI (Modified Plessey) instance. The checksum is added automatically
+    when building the bars.
+
+    :parameters:
+        code : int or bytes or string
+            Code MSI int without checksum (added automatically).
+            Code MSI bytes without checksum. requires byteorder for int conversion
+            Code MSI string without checksum. requires encoding and byteorder for int conversion
+        writer : barcode.writer Instance
+            The writer to render the barcode (default: SVGWriter).
+        byteorder : string
+            'big' or 'little' ; to convert bytes to int
+        encoding : string
+            if set, convert bytes to string and use this as a label. defaults to utf-8
+
+    limitations:
+        - only one check digit (Luhn Mod10)
+        - only standard prefix/suffix
+    """
+
+    name = "MSI/Modified Plessey"
+
+    def __init__(self, code, writer=None, byteorder=None, encoding=None):
```

Comment:
Type annotations missing.

Reply:
Can you please help me with this? I don't know how to do it when multiple types are allowed.

I have severe back and joint issues that are made much worse when sitting for extended periods of time, so I will not dig further into an obscure doc to find out.